doc/nvim-treesitter.txt

```diff
@@ -127,6 +127,10 @@ You can use |:TSInstall| `all` to install all parsers.
 :TSInstallInfo~
 
 List informations about currently installed parsers
+							           *:TSUpdate*
+:TSUpdate {language}~
+
+Update the installed parser of {language} or of all installed languages.
```

Comment:
```suggestion
Update the installed parser of {language} or all parsers if {language} is omitted.
```

Reply:
:+1: changed to `or all installed parsers if {language} is omitted`